File: app/src/main/java/com/skeleton/mvp/ui/onboarding/otpverification/OTPView.java
package com.skeleton.mvp.ui.onboarding.otpverification;


import com.skeleton.mvp.ui.base.BaseView;

/**
 * Developer: Geetanjali Gupta
 */
public interface OTPView extends BaseView {

    /**
     * On Verification Complete
     *
     * @param message Success Message
     */
    void onOtpVerificationSuccessful(final String message);

    /**
     * On Verification Complete
     *
     * @param message Success Message
     */
    void onResendOtpSuccessful(final String message);
}

File: app/src/main/java/com/skeleton/mvp/ui/onboarding/otpverification/OTPView.kt
package com.skeleton.mvp.ui.onboarding.otpverification;


import com.skeleton.mvp.ui.base.BaseView;

/**
 * Developer: Geetanjali Gupta
 */
interface OTPView : BaseView {

    /**
     * On Verification Complete
     *
     * @param message Success Message
     */
    fun onOtpVerificationSuccessful(message: String);

    /**
     * On Verification Complete
     *
     * @param message Success Message
     */
    fun onResendOtpSuccessful(message: String);
}
